installation/gcp/dm/firewall-template.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates the firewall."""
+
+
+def GenerateConfig(context):
+    """Creates the firewall"""
+    
+    resources = [
+        {
+            'name': f"allow-ssh-to-vmclarity-server-{context.env['deployment']}",
+            'type': 'compute.v1.firewall',
+            'properties': {
+                'network': '$(ref.' + context.properties['network'] + '.selfLink)',
+                'sourceRanges': ['0.0.0.0/0'],
+                'allowed': [{
+                    'IPProtocol': 'TCP',
```

Comment:
Should you enable also http/https access to the server?

Reply:
Not until we have RBAC, until then its the same as all the other providers, UI access is through the SSH tunnel